core/src/test/java/tech/beshu/ror/acl/blocks/rules/impl/GroupsProviderAuthorizationAsyncRuleTests.java

```diff
@@ -59,7 +60,7 @@ public void testUserAuthorizationSuccess() throws Exception {
     result = createRuleRunMatch(Lists.newArrayList("group1", "group3"),
         "/complex/groups", "POST",
         ImmutableMap.of("Content-Type", "application/x-www-form-urlencoded",
-            "Authorization", "Basic base64_encoded_token"),
```

Comment:
Why not to use the standard auth header "Authorization"?

Reply:
this is just a test to show even basic auth can be simply passed in one argument, no extra logic required. the header can be anything. it's just a name-value pair for users.